libyul/AsmJsonImporter.cpp

```diff
@@ -157,7 +157,12 @@ Literal AsmJsonImporter::createLiteral(Json::Value const& _node)
 	auto lit = createAsmNode<Literal>(_node);
 	string kind = member(_node, "kind").asString();
 
-	lit.value = YulString{member(_node, "value").asString()};
+	solAssert(member(_node, "hexValue").isString() || member(_node, "value").isString(), "");
+	if (_node.isMember("hexValue"))
```

Comment:
All literal members would now have `hexValue` member, wouldn't it? Why is this in an `if` branch?

Reply:
True!

Reply:
I mean this provides some kind of backwards-compatibility, so maybe why not?

Reply:
Okay. Then just rebasing and the suggestion on changelog should make this ready to merge.